test/promisepay_ex_test.exs

```diff
@@ -12,11 +12,12 @@ defmodule PromisepayExTest do
     )
 
     Config.filter_url_params(true)
-    Config.filter_request_headers("Authorization")
+    Config.filter_request_headers("basic_auth")
+    #Config.filter_sensitive_data("<PASSWORD>.+</PASSWORD>", "PLACEHOLDER")
```

Comment:
This can go away as we can always rely on commit history if ever we need to use this back again?